app/constants/authorities.js

```diff
@@ -3,3 +3,5 @@ export const PUBLIC_DATA_URL =
 
 export const AUTHORITIES_LIST_URL =
   'https://raw.githubusercontent.com/tripleblindmarket/safe-places/develop/healthcare-authorities.yaml';
+
+export const DEFAULT_NEWS_SITE_URL = 'https://covidsafepaths.org/in-app-news';
```

Comment:
since it now contains more than authorities url now, maybe refactor this class name to reflect what it contains...

Reply:
hrm. yeah. good point.

I think like in the other one I will leave this in the only file that uses it.. until it's used somewhere else.